kstreams/clients.py

```diff
@@ -16,37 +15,37 @@ class Consumer(aiokafka.AIOKafkaConsumer):
     def __init__(
         self,
         *args,
-        settings_prefix: str = "SERVICE_KSTREAMS_",
+        backend: Optional[Kafka] = None,
         key_deserializer: Optional[Callable] = None,
         **kwargs,
     ):
-        self.settings = PrefixedSettings(conf.settings, prefix=settings_prefix)
+        if backend is None:
```

Comment:
I am not really sure about this. It seems that `backend` should not be optional as at the moment of creating the engine, the `backend` should exist. Then, `Consumer` and `Producers` will receive the `backend` from the engine

Reply:
I thought about it. If we place it at the `create` level. It means it will be mandatory to set a `backend` if you use `Consumer` or `Producer` directly.